### ISSUE TITLE
NVDA does not announce placeholder of the quick input box

### ISSUE THREAD

**Issue by sandy081** [Microsoft/vscode#57501]
- Using NVDA
- Open the language mode quick picker from the status bar

- NVDA does not announces the placeholder text

**Comment by chrmarti** [comment]
Good catch! That broke with setting `aria-activedescendant`.
